images/azure/rocky-91.yaml

```diff
@@ -4,9 +4,13 @@ download_images: true
 packer:
   distribution: "rockylinux-9"                # Offer
   distribution_version: "rockylinux-9"  # SKU
+  # Azure Rocky linux official image: https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/erockyenterprisesoftwarefoundationinc1653071250513.rockylinux-9
   image_publisher: "erockyenterprisesoftwarefoundationinc1653071250513"
   image_version: "latest"
   ssh_username: "azureuser"
+  plan_image_sku: "rockylinux-9" # SKU
```

Comment:
Azure plan information needed to use marketplace image. 
Packer throwing following error when plan information was not provided. 
`Creating a virtual machine from Marketplace image or a custom image sourced from a Marketplace image requires Plan information in the request`

Reply:
This thread has lots of useful information https://forums.rockylinux.org/t/azure-rocky-image-on-marketplace/5230/46

Reply:
Thank you for looking it up